packages/superset-ui-plugin-chart-icicle-event/src/utils/IcicleEventTreeHelpers.ts

```diff
@@ -32,14 +32,14 @@ export function findDepth(node: IcicleEventNode, depth: number = 0): number {
 
   const maxDepth = d3Max(node.children.map(child => findDepth(child, depth + 1)));
 
-  return maxDepth || depth;
+  return maxDepth ?? depth;
 }
 
 export function hierarchySort(
   a: HierarchyNode<IcicleEventNode>,
   b: HierarchyNode<IcicleEventNode>,
 ): number {
-  if (a && a.value && b && b.value) {
+  if (a?.value && b && b.value) {
```

Comment:
why not `b?.value` as well?

Reply:
These are changed by `eslint` auto-fix. Perhaps it is not smart enough. I'll take a look.